HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -54,6 +68,7 @@ def self.apply_air_source_hvac_systems(model, cooling_system, heating_system,
       end
     elsif (heating_system.is_a? HPXML::HeatingSystem) && (heating_system.heating_system_type == HPXML::HVACTypeFurnace)
       obj_name = Constants.ObjectNameFurnace
+      max_pow_ratio_sch = nil
```

Comment:
This seems odd to me. Won't it prevent the EMS program when you have a variable speed AC with an attached furnace? If so, seems like we should replace the sample file that has a standalone variable speed AC with this HVAC configuration.

Reply:
This condition is for standalone furnace I think. So I added the test case for var speed ac+ attached furnace to make sure the cooling operation is added.